### PR TITLE
Update README.md for 0.2 release

### PR DESCRIPTION
**Pull request by pete-tempo**
Update README to reflect latest changes in 0.2 release and improve clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,40 +3,65 @@ A plugin to integrate ROS and Unreal via `rclcpp`.
...
-- ROS uses C++ features that are not enabled by default in Unreal C++ projects, including exceptions and RTTI (`dynamic_cast` and `type_id`). To resolve the former, **Unreal modules that depend on `TempoROS` must enable exceptions in their `Build.cs` file (use `bEnableExceptions = true;`).** For the latter, we've modified `rclcpp` to remove any uses of RTTI from header files.
+// Add a subscription.
+ROSNode->AddSubscription<FString>("my_topic", TROSSubscriptionDelegate<FString>::CreateLambda([](const FString& Message)
+{
+    UE_LOG(LogTemp, Display, TEXT("%s"), *Message);
+}));
 
-The upshot of this all is: although it is possible, we don't recommend using `rclcpp` directly in your project's code, as there are many potential pitfalls. Instead, use the `TempoROS` wrappers and APIs described below.
+// Publish a message.
+ROSNode->Publish("my_topic", TEXT("Hello World!));
```
Comment (gio-tempo) [inline README.md:43]: Missing end quote on "Hello World!".

File: README.md [hunk 2]
```diff
@@ -140,9 +186,26 @@ ROS includes [tf2](https://wiki.ros.org/tf2), a library that makes subscribing a
...
+- Run the `roudi` server as a separate process on the same machine. `TempoROS` comes with a pre-build `roudi` (at `TempoROS/Source/ThirdParty/rclcpp/Binaries/Linux/iox-roudi`), but one from a pacakged ROS installation should also work. Note that you'll have to relax its compatibility check, with `iox-roudi -x minor`, as the one `TempoROS` linked against won't match a packaged ROS installation's exactly.
+
+## Packaging
+You can use `TempoROS` as part of a packaged game. You can find a convenient script to package the project with the recommended settings in [TempoSample](https://github.com/tempo-sim/TempoSample/blob/main/Scripts/Package.sh).
+
+To package an Unreal project with `TempoROS`, you must specify its custom stage copy handler by adding `CustomStageCopyHandler=TempoROSCopyHandler` to your `Config/DefaultGame.ini`. This allows the package process to correctly copy symbolic links in the `rclcpp` libraries on all platforms.
+
+## Known Issues
+- To run an Unreal packaged game with TempoROS on Windows, you must add the directory `<package_root>/UE/TempoSample/Plugins/Tempo/TempoROS/Source/ThirdParty/rclcpp/Binaries/Windows` to your `PATH` environment variable.
```
Comment (gio-tempo) [inline README.md:210]: If TempoROS is meant to be standalone, do you mean to reference TempoSample in this path?

**Review by gio-tempo — APPROVED** [review]
Approved with minor comments.

File: README.md (hunk 2 above)
Comment (pete-tempo) [reply README.md]: Good catch, changed to "YourProjectName"

**Pull request merged**